web/src/queries/storage/config-model.ts

```diff
@@ -252,6 +278,23 @@ export function useBoot(): BootHook {
   };
 }
 
+export type PartitionHook = {
+  delete: () => void;
+};
+
+// driveName, like "/dev/sda"
+// mountPath, like "/" or "swap"
```

Comment:
It makes sense to follow here the same syntax (jsdoc?) we are using all along the rest of the code.

Reply:
So this could be the first properly documented function in this file.